wyzesense2mqtt/wyzesense.py

```diff
@@ -226,71 +231,115 @@ def AsyncAck(cls, cmd):
 
 
 class SensorEvent(object):
-    def __init__(self, mac, timestamp, event_type, event_data):
+    def __init__(self, mac, timestamp, event_type, event_data, pkt=None):
```

Comment:
This will let us pass the payload to the logging easier in the event of adding new devices in the future.